core/domain/summary_services.py

```diff
@@ -50,6 +61,9 @@ def get_displayable_exp_summary_dicts_matching_ids(exploration_ids):
                 'status': exploration_summary.status,
                 'ratings': exploration_summary.ratings,
                 'community_owned': exploration_summary.community_owned,
+                'contributors_summary':
```

Comment:
Maybe call this human_readable_contributors_summary to differentiate from the similarly named field on the exploration_summary instance.